java/client/src/main/java/glide/api/models/commands/RedisScoreLimit.java

```diff
@@ -0,0 +1,50 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.models.commands;
+
+public class RedisScoreLimit {
```

Comment:
I named it RedisScoreLimit because I don't use that class. I am using the interface to be able to achieve unions with the scoreLimit interface don't know another way to do this in java.

Reply:
Let's call it `ScoreLimitOptions` to be consistent with our other classes. 
We should consider moving the interface to a separate class (`RedisOptions` or similar).